docs/admin/releases/v0.8.0-alpha.7.mdx

```diff
@@ -0,0 +1,128 @@
+* Collaboration server in admin-server [#85](https://github.com/contember/admin/pull/85)
+* Editor: Sortable blocks and major refactoring including new API for adding custom blocks and bug fixes [#71](https://github.com/contember/admin/pull/71), [#76](https://github.com/contember/admin/pull/76), [#82](https://github.com/contember/admin/pull/82)
+* Datagrid: `HasOneSelectCell` and `HasManySelectCell` [#87](https://github.com/contember/admin/pull/87)
+* Datagrid: Remembers selected filters and page when navigating back [#81](https://github.com/contember/admin/pull/81)
+
+
+### Fixes
+* Various select field fixes [#82](https://github.com/contember/admin/pull/82), [#83](https://github.com/contember/admin/pull/83)
+* `LogoutLink` in Safari [#71](https://github.com/contember/admin/pull/71)
+
+
+**Full Changelog**: https://github.com/contember/admin/compare/v0.7.0...v0.8.0-alpha.7
+
+## Migration
+
+We've simplified the layout components to make them easier to use. To use the new layout, you need to update a few things.
+
+### Mandatory changes
+
+#### Rename `<Layout />` props 
+
+- `topStart` to `sidebarHeader`
+- `topEnd` to `sidebarFooter`
+- `sideBar` to `navigation`
+
+<CompareCode>
+
+```tsx {2-4}
+<Layout
+	topStart={...}
+	topEnd={...}
+	sideBar={...}
+/>
+```
+
+```tsx {2-4}
+<Layout
+	sidebarHeader={...}
+	sidebarFooter={...}
+	navigation={...}
+/>
+```
+
+</CompareCode>
+
+#### Remove `<TitleBar />`
+
+- Move prop `actions` from the `<TitleBar />` component to page component (e.g. `<Page />`, `<GenericPage />`, `<LayoutPage />`).
+- Remove the `<TitleBar />` component and insert the contents of the `<TitleBar />` component into the `title` property of the page.
+
+<CompareCode>
+
+```tsx {4-8}
+<GenericPage 
+	pageName="blogList"
+>
+	<TitleBar 
+		actions={...}
+	>
+		Article
+	</TitleBar>
+	...
+</GenericPage>
+```
+
+```tsx {3-4}
+<GenericPage 
+	pageName="blogList"
+	title="Article"
+	actions={...}
+>
+	...
+</GenericPage>
+```
+
+</CompareCode>
+
+#### Rename `<LayoutInner />` and remove `<LayoutSide />`
+
+- Change the `<LayoutInner />` to `<LayoutPage />`.
+- Remove the `<LayoutSide />` component and insert the content into the `side` prop on `<LayoutPage />`.
+
+<CompareCode>
+
+```tsx {2,4-7}
+<LayoutContent>
+	<LayoutInner>
+	...
+	</LayoutInner>
+	<LayoutSide>
+		<PersistButton />
+	</LayoutSide>
+</LayoutContent>
+```
+
+```tsx {2,4}
+<LayoutContent>
+	<LayoutPage side={<PersistButton />}>
+	...
+	</LayoutPage>
+</LayoutContent>
+```
+
+</CompareCode>
+
+#### Remove `<AltPersistButton />`
+
+- We no longer support `<AltPersistButton />`. Remove it or change it to `<PersistButton />`.
+
+#### Tenant pages
+
+- Change the `<GenericPage />` that wraps the tenant pages to `<Page />`.
+
+### Recommended changes
+
+Remove unnecessary component `<Box />`.  We added a new component `<Section />` which should be used instead of `<Box />` to define groups of fields.
```

Comment:
There few things to be noted, here's my suggestion (maybe in bullets for better readability):

 "... split groups of fields on the page into logical page sections. Sections with titles are also listed within the page tabs navigation. Sections are not meant to be nested within other sections and should only be used to define top level sections of the page. Sections are required to be parent elements to dimensions not the other way around due to styling limitations."